internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -0,0 +1,431 @@
+		usrDefault := dbUsername()
+		if usrDefault != "" {
+			message = fmt.Sprintf("Insert the Username for authenticating to MongoDB [Press Enter to use '%s']", usrDefault)
+		}
+		q := &survey.Question{
+			Name: "dbUsername",
+			Prompt: &survey.Input{
+				Message: message,
+				Help:    usage.DBUsername,
+				Default: usrDefault,
+			},
+			Validate: func(val interface{}) error {
+				username, _ := val.(string)
+				user, err := opts.store.DatabaseUser(convert.AdminDB, opts.ConfigProjectID(), username)
+				if err != nil {
+					if !strings.Contains(err.Error(), fmt.Sprintf("No user with username %s exists.", username)) {
+						return err
+					}
+				}
+
+				if user != nil {
+					return errors.New("a user with this username already exists")
+				}
+
+				return nil
+			},
+		}
+
+		qs = append(qs, q)
+	}
+
+	if opts.DBUserPassword == "" {
+		q := &survey.Question{
+			Name: "dbUserPassword",
+			Prompt: &survey.Password{
+				Message: "Insert the Password for authenticating to MongoDB [Press Enter to use an auto-generated password]",
+				Help:    usage.Password,
+			},
+		}
+
+		qs = append(qs, q)
+	}
+
+	return qs
+}
+
+func (opts *Opts) newAccessListQuestion() *survey.Question {
+	if opts.IPAddress != "" {
+		return nil
+	}
+
+	message := "Insert the IP entry to add to the Access List"
+	//nolint:ifshort //false positive
+	publicIP := net.IPAddress()
+
+	if publicIP != "" {
+		message = fmt.Sprintf("Insert the IP entry to add to the Access List [Press Enter to use your public IP address '%s']", publicIP)
+	}
+
+	return &survey.Question{
+		Name: "ipAddress",
+		Prompt: &survey.Input{
+			Message: message,
+			Help:    usage.AccessListIPEntry,
+			Default: publicIP,
+		},
+	}
+}
+
+func (opts *Opts) newClusterQuestions() []*survey.Question {
+	var qs []*survey.Question
+
+	if opts.ClusterName == "" {
+		clusterName := opts.newClusterName()
+		message := "Insert the cluster name"
+		if clusterName != "" {
+			message = fmt.Sprintf("Insert the cluster name [Press Enter to use the auto-generated name '%s']", clusterName)
+		}
+		q := &survey.Question{
+			Name: "clusterName",
+			Prompt: &survey.Input{
+				Message: message,
+				Help:    usage.ClusterName,
+				Default: clusterName,
+			},
+		}
+		qs = append(qs, q)
+	}
+
+	if opts.Provider == "" {
+		q := &survey.Question{
+			Name: "provider",
+			Prompt: &survey.Select{
+				Message: "Insert the cloud service provider on which Atlas provisions the hosts",
+				Help:    usage.Provider,
+				Options: []string{"AWS", "GCP", "AZURE"},
+			},
+		}
+		qs = append(qs, q)
+	}
+
+	return qs
+}
+
+func (opts *Opts) newRegionQuestions() *survey.Question {
+	if opts.Region != "" {
+		return nil
+	}
+	return &survey.Question{
+		Name: "region",
+		Prompt: &survey.Select{
+			Message: "Insert the physical location of your MongoDB cluster",
+			Help:    usage.Region,
+			Options: DefaultRegions[strings.ToUpper(opts.Provider)],
+		},
+	}
```

Comment:
[nit] let's follow. asimilar pattern to the config command where we moved any `survey.Prompt` or `survey.Question
` to a `prompt.go` file, it'll make this easier to review